### PR TITLE
sugardaddy

### PR DESCRIPTION
**Pull request by OuChiaYun**
AAA 甚麼先別管

### CODE REVIEW

File: packages/contracts/contracts/Info.sol [hunk 1]
```diff
@@ -41,6 +41,13 @@ contract INoneMoney {
     event give_money(bool success_holder);
     event return_money(bool success_donor);
 
+    struct SugarDaddy {
+        bool exit;
```
Comment (ntut-xuan) [inline packages/contracts/contracts/Info.sol:45]:
```suggestion
        bool exist;
```

**Review by ntut-xuan — CHANGES_REQUESTED** [review]
LGTM.

File: packages/contracts/contracts/Info.sol (hunk 1 above)
Comment (OuChiaYun) [reply packages/contracts/contracts/Info.sol]: fixed

**Review by ntut-xuan — APPROVED** [review]
LGTM. You are ready to go!

**Pull request merged**